### PR TITLE
Color theme ao

### PR DESCRIPTION
**Pull request by YardQuit**
Formulated a dark color scheme evoking the essence of Fedora Linux, incorporating hues of blue reminiscent of the brand's identity.

### CODE REVIEW

File: runtime/themes/ao.toml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+# SYNTAX HIGHLIGHTING
+"attribute" = { fg = "yellow" }                                                                      #
+"type" = { fg = "white" }                                                                            # types e.g. return type: Result, String, etc
+"type.builtin" = { fg = "white" }                                                                    # primitive types provided by the language (int, usize)
+"type.parameter" = { fg = "white" }                                                                  # generic type parameter (T)
+"type.enum" = { fg = "white" }                                                                       #
+"type.enum.variant" = { fg = "white" }                                                               #
+"constructor" = { fg = "orange" }                                                                    # e.g. Ok, Err, etc
+"constant" = { fg = "blue" }                                                                         # const name 
+"constant.builtin" = { fg = "blue" }                                                                 #
```
Comment (archseer) [inline runtime/themes/ao.toml:13]: Quite a lot of these seem redundant. More specific scopes will always fall back on the more general scopes so if constant is already blue there's no need to declare constant.builtin that's also blue

Comment (YardQuit) [reply runtime/themes/ao.toml]: Thanks for your comment. I have updated it by commenting out what I then believed to be redundant settings. If confirmed, I can remove all commented lines to clean it up.

Comment (archseer) [reply runtime/themes/ao.toml]: Looks correct! Similar thing applies for the other theme, I think your other PR https://github.com/helix-editor/helix/pull/10381 leaked into this one

Comment (YardQuit) [reply runtime/themes/ao.toml]: Thanks, I have removed redundancy from both themes

**Comment by YardQuit** [comment]
Sorry for my ignorance in merge requests, but is there anything else expected from my end?
Same question goes for https://github.com/helix-editor/helix/pull/10381

**Comment by YardQuit** [comment]
Thanks, but this just became to complicated to me. I don't really know what to do or why this is in conflict with https://github.com/helix-editor/helix/pull/10381 or how to solve it without messing things up. Any advice on the best and easiest approach?

**Comment by the-mikedavis** [comment]
You can merge this branch with the current Helix master. It might be easier to create a new branch off of the current Helix master though and add the new theme file to it and create a new PR.

**Comment by YardQuit** [comment]
Yes, I will give it a try that, thanks! :)

**Comment by the-mikedavis** [comment]
Closing in favor of #11063

**Pull request closed without merging**